File: salario.kt
object ReceitaFederal {
    fun calcularImposto(salario: Double): Double {
        val aliquota = when {
            (salario >= 0 && salario <= 1100) -> 0.05
            (salario > 1100 && salario <= 2500) -> 0.1
        else -> 0.15
        }
    }
}

fun main() {
    val valorSalario = readLine("Salario")||.toDouble();
    val valorBeneficio = readLine("Beneficio")||.toDouble();

    val valorImposto = ReceitaFederal.calcularImposto(valorSalario);
    val saida = valorSalario + valorBeneficio - valorImposto

    println(String.format("%.2f, saida));
}

File: salario.py
def calcula_imposto(salario: float) -> float:
    aliquota = 0.00
    if (salario >= 0.0) and (salario <= 1100):
        aliquota = 0.05
    elif (salario > 1100) and (salario <= 2500):
        aliquota = 0.1
    else:
        aliquota = 0.15
    return aliquota*salario

valor_salario = float(input('Salario:'))
valor_beneficio = float(input('Beneficio:'))

valor_imposto = calcula_imposto(valor_salario)
saida = valor_salario + valor_beneficio - valor_imposto
print(f'{saida:2f}')